app/contexts/media-context.tsx

```
@@ -0,0 +1,220 @@
+        setVolume: handleVolumeChange,
+        seekTo,
+      }}
+    >
+      <audio
+        ref={audioRef}
+        onTimeUpdate={() => setProgress(audioRef.current?.currentTime || 0)}
+        onEnded={() => setIsPlaying(false)}
+        onLoadStart={() => setIsLoading(true)}
+        onCanPlay={() => setIsLoading(false)}
+        onError={() => setIsLoading(false)}
+      />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)